model_path: /home/luc/Desktop/4class_googlenet/deploy.prototxt
weights_path: /home/luc/Desktop/4class_googlenet/model.caffemodel
mean_path: /home/luc/Desktop/4class_googlenet/mean.binaryproto
label_path: /home/luc/Desktop/4class_googlenet/labels.txt

image_topic: /camera/rgb/image_raw/downsized
pub_topic: /caffe_ret

n_multi_hypothesis: 4